nasa-gibs/worldview
# WV-3058

## Description
The close button on modals aren't right justified anymore - they should be moved back to the right.

Fixes #WV-3058

## How To Test

- git checkout wv-3058
- npm run watch
- Confirm the close buttons on various modals are on the right side

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
